src/header-components/_header.scss

```diff
@@ -0,0 +1,29 @@
+.header {
```

Comment:
Again, killing it with Sass nesting on this! I't super helpful for the organization here